src/components/Footer/Footer.js

```diff
@@ -1,5 +1,5 @@
-import * as React from 'react';
 import PropTypes from 'prop-types';
+import * as React from 'react';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import order has been changed. While this doesn't affect the functionality, it's generally a good practice to keep a consistent order of imports for better readability and maintainability. In this case, it seems like the original order was more logical, with React being imported first.
```diff
- import PropTypes from 'prop-types';
- import * as React from 'react';
+ import * as React from 'react';
+ import PropTypes from 'prop-types';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->